project/build.scala

```diff
@@ -114,6 +114,7 @@ object RhoBuild extends Build {
         license,
 
         libraryDependencies <++= scalazVersion { sz => Seq(
+          shapeless,
```

Comment:
Do we need the explicit dependency or can we use the transient one from http4s to avoid compatibility problems?